app/src/main/java/com/nervousfish/nervousfish/activities/BluetoothConnectionActivity.java

```diff
@@ -228,14 +226,17 @@ public void stopDiscovering() {
     }
 
     /**
-     * Called when a new bluetooth device is connected
+     * Called when the device is connected over Bluetooth
      *
-     * @param event Details about the event
+     * @param event Describes the event
      */
-    @Subscribe
+    @Subscribe(threadMode = ThreadMode.MAIN)
     public void onEvent(final BluetoothConnectedEvent event) {
```

Comment:
But isn't it kinda nice to know that each class actually received the event? For debugging it should not be a problem if all of them log (even if the messages are identical, because you can see what class did the logging)